web/models/meeting_type.ex

```diff
@@ -7,14 +7,16 @@ defmodule Meetings.MeetingType do
     field :title, :string
     field :description, :string
     field :location, :string
+    field :building, :string
```

Comment:
The building information was part of location, but sometimes that doesn't work in google maps, so I took it off and put it in it's own field. Most of the locations have building information, that's why it's not in extras.